<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DriverPower</class>
 <widget class="QWidget" name="DriverPower">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>652</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>652</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>652</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:transparent;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <item row="5" column="1" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="sign">
       <property name="minimumSize">
        <size>
         <width>1000</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/背景_标题2.png);
font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 28px;
color: #FFFFFF;</string>
       </property>
       <property name="text">
        <string>驱动电机</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_back">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#btn_back{
background-image: url(:/按键_上一页2.png);
border:none;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QWidget" name="widget_5" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>210</width>
       <height>210</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/显示_参数2.png);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <widget class="QLabel" name="driver_control_temp">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 60px;
color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>0000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_16">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 25px;
color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>驱动电机控制
器温度(摄氏度)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>210</width>
       <height>210</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/显示_参数2.png);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="direct_current">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 60px;
color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>0000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 25px;
color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>直流电流(A)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QWidget" name="widget_6" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>210</width>
       <height>210</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/显示_参数2.png);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <item>
       <widget class="QLabel" name="driver_temper">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 60px;
color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>0000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_17">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 25px;
color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>驱动电机温度
(摄氏度)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QWidget" name="widget_3" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>210</width>
       <height>210</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/显示_参数2.png);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QLabel" name="real_torque">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 60px;
color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>0000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 25px;
color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>实际转矩(Nm)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="4">
    <widget class="QWidget" name="widget_4" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>210</width>
       <height>210</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/显示_参数2.png);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QLabel" name="emc_input_v">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 60px;
color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>0000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_15">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 25px;
color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>电机控制器
输入电压(V)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="5">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>210</width>
       <height>210</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/显示_参数2.png);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="real_rev">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 60px;
color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>0000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 25px;
color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>实际转速(rpm)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3" colspan="2">
    <widget class="QWidget" name="widget_7" native="true">
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="Label">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 35px;
color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>工作模式</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="work_mode">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">border-color: #FFFFFF;
font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 35px;
color: #FFFFFF;
background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="Label2">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 35px;
color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>运行状态</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="running_state">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-color: #FFFFFF;
font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 35px;
color: #FFFFFF;
background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="Label3">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 35px;
color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>故障状态</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="falut_state">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-color: #FFFFFF;
font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 35px;
color: #FFFFFF;
background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="Label4">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 35px;
color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>电机转向</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="em_direction">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-color: #FFFFFF;
font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 35px;
color: #FFFFFF;
background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
